README.md

```diff
@@ -0,0 +1,63 @@
+## Cloudify ARIA Extensions
+
+This repository enables ARIA to utilize some of Cloudify's abilities, such as orchestrating using AWS and Openstack.
+
+### Installing
+
+1. clone the repository
+
+`git clone https://github.com/cloudify-cosmo/cloudify-aria-extensions.git`
+
+2. install the requirements:
+
+`pip install -r requirements.txt`
+
+3. install Cloudify ARIA extensions:
+
+`pip install .`
+
+4. (optional, for developing purposes) install the test requirements:
+
+`pip install -r aria_extension_tests/requirements.txt`
+
+#### An example for using a Cloudify plugin with ARIA:
+
+We will use the Cloudify AWS Plugin as an example to show how to utilize a Cloudify plugin using ARIA. Currently, ARIA supports the [Cloudify AWS Plugin](https://github.com/cloudify-cosmo/cloudify-aws-plugin) and the [Cloudify Openstack Plugin](https://github.com/cloudify-cosmo/cloudify-openstack-plugin).
+
+We'll need a `.wgn` file of the AWS plugin. This file can be obtained in two ways be done in two methods. The First:
```

Comment:
1) link to centos/rhel built wagons directly, and before the "build wagon from source"
2) fix downloads page link
3) have in the readme which versions of the plugins have been tested
4) add a short line explaining newer versions should work as well and users are welcome to try and build wagons from source